test/test_tracer_telemetry.cpp

```diff
@@ -54,7 +54,7 @@ TEST_CASE("Tracer telemetry", "[telemetry]") {
       auto app_started = nlohmann::json::parse(app_started_message);
       REQUIRE(is_valid_telemetry_payload(app_started) == true);
       REQUIRE(app_started["request_type"] == "message-batch");
-      REQUIRE(app_started["payload"].size() == 2);
+      REQUIRE(app_started["payload"].size() == 1);
```

Comment:
Can you explain to me how this changed from `2` to `1` with your changes?

Reply:
This [PR](https://github.com/DataDog/dd-trace-cpp/commit/963f78b0909b2020e348cd6e704249ee7cbafcd2#diff-0f62c1beabc439887a3fa87039163c77efd2589ad2e462616941940d755f0c3b) changes from `1` to `2` because it would set a default `integration_name` to `datadog`. I reverted this behaviour.